### PR TITLE
feat: add ci workflow

### PR DESCRIPTION
**Pull request by mandarini**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -6,10 +6,11 @@ on:
   pull_request:
 
 permissions:
+  actions: read
   contents: read
 
 jobs:
-  tests:
```
Comment (meeroslav) [inline .github/workflows/ci.yml:12]: Why this change?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -29,7 +33,10 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: 'pnpm'
 
-      # Cargo already skips downloading dependencies if they already exist
```
Comment (meeroslav) [inline .github/workflows/ci.yml:32]: Leave this

File: crates/node/project.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "targets": {
```
Comment (meeroslav) [inline crates/node/project.json:2]: Let's move this to package.json perhaps?

File: integrations/package.json [hunk 4]
```diff
@@ -2,6 +2,9 @@
   "name": "internal-integrations",
   "version": "0.0.0",
   "private": true,
+  "scripts": {
+    "integration-test": "vitest --no-file-parallelism"
```
Comment (meeroslav) [inline integrations/package.json:6]: Was this automatically added?

**Review by meeroslav — CHANGES_REQUESTED** [review]

**Comment by mandarini** [comment]
Moved to here: https://github.com/mandarini/tailwindcss/pull/5

File: crates/node/project.json (hunk 3 above)
Comment (mandarini) [reply crates/node/project.json:2]: done

File: integrations/package.json (hunk 4 above)
Comment (mandarini) [reply integrations/package.json:6]: no juri and i added it
